packages/dashmate/configs/schema/configJsonSchema.js

```diff
@@ -243,9 +243,29 @@ module.exports = {
           additionalProperties: false,
           required: ['name', 'minimumDifficultyBlocks', 'powTargetSpacing'],
         },
-        debug: {
-          type: 'integer',
-          enum: [0, 1],
+        log: {
+          type: 'object',
+          properties: {
+            file: {
+              type: 'object',
+              properties: {
+                level: {
```

Comment:
Let's call it `categories` and it must be `enum` with `all` and other categories